src/plugile.rpgle

```diff
@@ -112,7 +112,8 @@
      D search00        s             20A   inz(*BLANKS)
      D pLook00         s               *   inz(%addr(search00))
      D myLook00        ds                  likeds(over_t) based(pLook00)
-
+      * flag for 32-bit or 64-bit
+     D xBit            s             20u 0 inz(4)
```

Comment:
This is confusing if the flag is called xBit, but the value is in bytes - should convert from bytes to bits (ie. 32 or 64)

Reply:
Also why the x? I'd call it `paseBits` or something.

Reply:
Agree. I changed it to prtSize.

Reply:
I still think it makes more sense in bits. Right now there's comments all over to explain that the pointer size 4 means 32-bit or pointer size 8 means 64-bit, eg. 

```text
if getXBit() = 4;   // 32-bit
```

If it was in bits already, we wouldn't need the comment as it would be clear from the if check itself:
```text
if paseBits = 32;
```


It's a simple enough change:
```
tBit = Qp2ptrsize();
```
becomes
```
tBit = Qp2ptrsize() * 8;
```

Reply:
OK. I will change back to use bits. But not sure we should call it paseBits as DB2 call goes this path too. Maybe call it ptrBits.

Reply:
Or how about ptrBytes? since Qp2ptrsize() return the size in bytes.

Reply:
It doesn't really matter what that returns. We can use what makes most sense for us, which is bits.